src/System.Runtime.Numerics/ref/System.Runtime.Numerics.cs

```diff
@@ -153,14 +153,15 @@ namespace System.Numerics
         public static System.Numerics.BigInteger Remainder(System.Numerics.BigInteger dividend, System.Numerics.BigInteger divisor) { throw null; }
         public static System.Numerics.BigInteger Subtract(System.Numerics.BigInteger left, System.Numerics.BigInteger right) { throw null; }
         public byte[] ToByteArray() { throw null; }
+        public byte[] ToByteArray(bool isUnsigned=false, bool isBigEndian=false) { throw null; }
```

Comment:
How come the other methods added optional parameters to the existing methods, but here you are adding a new overload?  Why not change `ToByteArray()`?

Reply:
See the API proposal issue.  `ToByteArray()` has already shipped, and cannot be altered.  The other, Span-based methods, are new for netcoreapp21, so they're still flexible.